src/frontend/src/helpers/Storage/LocalStorage.ts

```diff
@@ -193,22 +219,21 @@ class LocalStorage implements AbstractStorage {
     });
   }
 
-  public async newFile(proj: ProjectID,
+  public async newFile(pid: ProjectID,
                        name: string,
-                       contents?: string,
```

Comment:
contents is undefined when a file is unavailable offline (i.e a binary file); note newFile is called in syncAll which has to deal with this.